app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<p>Here you will soon see the list of all users</p>
\ No newline at end of file
+<% @users.each do |user| %>
```

Comment:
- [OPTIONAL] 📘 **Suggestion**: If you haven’t considered it already, using partials like `app/views/users/_user.html.erb` to render users is a best practice, making the code more modular and manageable. Here's how:
```erb
<%= render @users %>